polyfill/lib/absolute.mjs

```diff
@@ -139,6 +139,10 @@ export class Absolute {
     let resultString = `${year}-${month}-${day}T${hour}:${minute}${seconds ? `:${seconds}` : ''}${timeZoneString}`;
     return resultString;
   }
+  toJSON() {
+    if (!ES.IsAbsolute(this)) throw new TypeError('invalid receiver');
+    return this.toString('UTC');
```

Comment:
We should not call other methods on `this`; please move the formatting parts into an abstract op.